src/api-reference/financial-integration/v4.Financial-Integration-Use-Cases-Extract-Information.markdown

```diff
@@ -5,7 +5,7 @@ layout: reference
 
 {% include prerelease.html %}
 
-**Note:** The extract file excerpts in this document are a reference point for those developers who have used those files for prior financial integrations. This excerpt is intended to help those developers transition from an extract file to the Financial Integration Service. This allows you to identify what you have been using from the Extract File and to locate the same data in the new JSON response. If you have not used the extract file previously, the Extract file excerpts can be skipped.
```

Comment:
I would prefer we just remove the word 'Note' altogether -- I'm unsure of the purpose.

Reply:
In this case, we want to warn readers that this information is only for a subset of readers. If people who aren't familiar with the extracts read this content, it will only confuse them. So, I'm open to something like Important instead of Note, it can be anything but we need it to be eye-catching.

Reply:
Reverted.